Assets/Scripts/Board/Graph.cs

```diff
@@ -80,17 +97,47 @@ public class TileNode : Node
 
     public Graph() { }
 
+    public static List<VertexDirection> GetVertexDirectionNeighbors(VertexDirection vertexDirection)
+    {
+        return vertexDirection switch
+        {
+            VertexDirection.North => new() { VertexDirection.Northwest, VertexDirection.Northeast },
+            VertexDirection.Northeast => new() { VertexDirection.North, VertexDirection.Southeast },
+            VertexDirection.Southeast => new() { VertexDirection.Northeast, VertexDirection.South },
+            VertexDirection.South => new() { VertexDirection.Southeast, VertexDirection.Southwest },
+            VertexDirection.Southwest => new() { VertexDirection.South, VertexDirection.Northwest },
+            VertexDirection.Northwest => new() { VertexDirection.Southwest, VertexDirection.North },
+            _ => new() { },
+        };
+    }
+
+    public static Vector3 GetVertexOffsetVector(VertexDirection vertexDirection)
+    {
+        return vertexDirection switch
+        {
+            VertexDirection.North => new Vector3(0, 1f / Mathf.Sqrt(3), 0),
+            VertexDirection.Northeast => new Vector3(1f / 2, 1f / (2 * Mathf.Sqrt(3)), 0),
+            VertexDirection.Southeast => new Vector3(1f / 2, -1f / (2 * Mathf.Sqrt(3)), 0),
+            VertexDirection.South => new Vector3(0, -1f / Mathf.Sqrt(3), 0),
+            VertexDirection.Northwest => new Vector3(-1f / 2, 1f / (2 * Mathf.Sqrt(3)), 0),
+            VertexDirection.Southwest => new Vector3(-1f / 2, -1f / (2 * Mathf.Sqrt(3)), 0),
+            _ => Vector3.zero,
+        };
+    }
+
```

Comment:
I wonder if we can make both of these just an inline dictionary instead of a function? is that more or less efficient? would be cleaner, and we technically shouldn't need to ever have a default Zero return from a dictionary

Reply:
I was debating between a switch vs a dictionary. Hmm, yeah, I think a dictionary might be cleaner.